coralproject/talk
# [CORL-1056] Configurable RTE

## What does this PR do?
- Implement RTE configuration
- Prevent empty comments from being accepted by the server (e.g. only empty lines)

## How do I test this PR?
- Access config in Admin -> Config -> General -> Rich-text comments
- See result on stream and post comments

Requires https://github.com/coralproject/talk/pull/2956
